neoload/commands/fastfail.py

```diff
@@ -0,0 +1,155 @@
+import click
+
+from commands import test_settings, test_results
+from neoload_cli_lib import user_data, tools, displayer, running_tools
+from neoload_cli_lib import tools
+import logging
+import yaml
+from datetime import datetime, date
+import time
+import sys
+import traceback
+from dateutil.relativedelta import relativedelta
```

Comment:
The is some imports that are never used... Can you clean up a bit ?

Reply:
Resolved with 11d9f37